src/components/reusables/Tabs.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+
+export interface TabsProps {
+  items: {
+    [key: string]: JSX.Element;
+  };
+}
+
+export interface TabsState {
+  tab: string;
+}
+
+export default class Tabs extends React.Component<TabsProps, TabsState> {
+  constructor(props: TabsProps) {
+    super(props);
+    this.select = this.select.bind(this);
+    this.makeTabs = this.makeTabs.bind(this);
+    this.state = { tab: Object.keys(this.props.items)[0] };
+  }
+
+  select(e: __React.SyntheticEvent) {
+    let tabName = (e.currentTarget as HTMLElement).innerText;
+    this.setState({ tab: tabName });
+  }
+
+  makeTabs(): Array<Array<JSX.Element>> {
+    let navs = [] as Array<JSX.Element>;
+    let content = [] as Array<JSX.Element>;
+    Object.entries(this.props.items).map(item => {
+      let hidden = item[0] === this.state.tab ? "" : "hidden";
+      let current = item[0] === this.state.tab ? "current" : "";
+      let navItem = (
+        <li key={item[0]} className={`tab-nav ${current}`}>
+          <button className="btn btn-default" onClick={this.select}>
```

Comment:
This button needs a role of "tab" as well as keeping track if it is selected or not with `aria-selected`.

Reply:
This also needs an id for another issue down below.